edge/pkg/metamanager/process.go

```diff
@@ -255,6 +267,12 @@ func (m *metaManager) processUpdate(message model.Message) {
 	case cloudmodules.PolicyControllerModuleName:
 		resp := message.NewRespByMessage(&message, OK)
 		sendToCloud(resp)
+	case modules.MetaManagerModuleName:
```

Comment:
It may cause confused using `metamanager` module here because we already have a module named `meta`. How about name this module  `metaserver` because it seems the request are from metaserver?   but metaserver is not an independent module, I'm not sure if metaserver is suitable. @fisherxu @luomengY @WillardHu @wbc6080

Reply:
> It may cause confused using `metamanager` module here because we already have a module named `meta`. How about name this module `metaserver` because it seems the request are from metaserver? but metaserver is not an independent module, I'm not sure if metaserver is suitable. @fisherxu @luomengY @WillardHu @wbc6080

I think using a `metaserver` is possible, as the source here comes from the `metaserver`.

Reply:
> It may cause confused using `metamanager` module here because we already have a module named `meta`. How about name this module `metaserver` because it seems the request are from metaserver? but metaserver is not an independent module, I'm not sure if metaserver is suitable. @fisherxu @luomengY @WillardHu @wbc6080

@WillardHu @fisherxu

Reply:
> I think using a `metaserver` is possible, as the source here comes from the `metaserver`.
+1

Reply:
OK, I added and registered `metaserver` module, and set the msgsource to `metaserver`.